Luke Smith Portfolio.css

```diff
@@ -0,0 +1,106 @@
+body {
+	margin: auto;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.topnav {
+  overflow: auto;
+  background-color: white;
+  position: fixed;
+}
+
+.topnav a {
+  float: left;
+  color: black;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+  font-size: 17px;
+}
+
+.topnav a:hover {
+  background-color: #ddd;
```

Comment:
I'm not sure this is formatted correctly. A hex code is usually six. digits i.e. #dddddd for a very light grey.

You also seem to be using hex codes and the predefined colours together - I think best practice style wise is to stick to just one of them where possible.

White is #ffffff
Black is #000000

Reply:
Had a look online and 3-digit colour codes will work. I'd recommend consistent formatting though - if you're using them then use only them.